fitlins/workflows/base.py

```diff
@@ -85,6 +85,8 @@ def init_fitlins_wf(database_path, out_dir, analysis_level, space,
         iterfield=['session_info', 'bold_file'],
         name='design_matrix')
 
+    model.inputs.drift_model = drift_model
```

Comment:
```suggestion
    design_matrix.inputs.drift_model = drift_model
```

Though you could also just make it:

```Diff
     design_matrix = pe.MapNode(
-        DesignMatrix(drop_missing=drop_missing),
+        DesignMatrix(drop_missing=drop_missing, drift_model=drift_model),
         iterfield=['session_info', 'bold_file'],
         name='design_matrix')
```